doc/how_to_guides/using_ompl_constrained_planning/ompl_constrained_planning.rst

```diff
@@ -0,0 +1,293 @@
+Using OMPL Constrained Planning
```

Comment:
Since this is in the HOW TO section consider renaming it into `How to use OMPL constrained planning?`